i_scene_cp77_gltf/main/bartmoss_functions.py

```diff
@@ -80,22 +94,31 @@ def getCustomProps(obj):
     
 # returns a list of modifiers for the given object
```

Comment:
there a reason not to just return obj.modifiers.keys()? thats a list of all the modifiers